auslib/web/admin/views/permissions.py

```diff
@@ -44,7 +44,8 @@ def get(self, username):
                                detail="You are not authorized to view permissions of other users.")
 
         permissions = dbo.permissions.getUserPermissions(username)
-        if not permissions:
+
+        if username != current_user and not permissions:
```

Comment:
Those tests are both correct, but keep in mind that even if someone _else_ was querying "/users/vikas", it should still return a 404. Put another way: the only thing that factors into the decision is whether or not the username being queried is "current" or not.

Does that make sense?